app/Http/Controllers/OrderController.php

```diff
@@ -13,15 +13,15 @@ public function create(){
 
     public function store(Request $request){
         $request->validate([
-            'rangkap' => 'required',
-            'tinta' => 'required',
-            'jenis_kertas' => 'required',
-            'jilid' => 'required',
-            'jasa_antar' => 'required',
-            'file' => 'required',
+            'rangkap' => 'required|integer|between:1,100',
+            'tinta' => 'required|max:50',
+            'jenis_kertas' => 'required|max:50',
+            'jilid' => 'required|max:100',
+            'jasa_antar' => 'required|boolean',
+            'file' => 'required|boolean|file|mimes:ppt,pptx,doc,docx,pdf,xls,xlsx|max:204800',
             'takenTime' => 'required',
-            'notes' => 'required',
-            'agreement' => 'required'
+            'notes' => 'required|min:3|max:1000',
+            'agreement' => 'accepted'
```

Comment:
What that's mean 'accepted'?

Reply:
Hii @alvinisasi according to laravel documentation accepted mean the field under validation must be yes, on, 1, or true.  You can read by your self [laravel doc](https://laravel.com/docs/7.x/validation#rule-accepted)